client/Character.h

```diff
@@ -14,8 +14,8 @@ class Character : Object {
 	GLuint EBO;
 
 	glm::mat4 model;
-	glm::mat4 view;
-	glm::mat4 projection;
+	glm::mat4 * view;

```

Comment:
view and projection matrices are changed to pointer, so in the future when we added camera tracking with player object, the view matrix are not copied and sent over for every frame